deploy/StableJoeStaking.js

```diff
@@ -38,14 +38,8 @@ module.exports = async function ({ getNamedAccounts, deployments }) {
         proxyContract: "OpenZeppelinTransparentProxy",
         execute: {
           init: {
-            methodName: "initialize",
-            args: [
-              rewardToken,
-              joeAddress,
-              feeCollector,
-              depositFeePercent,
-              smolJoes,
-            ],
+            methodName: "initializeSmolJoes",
+            args: [smolJoes],
```

Comment:
This should still call `initialize`, the `initializeSmolJoes` is only for an upgrade, and we need to call it by the multisig